sky/serve/controller.py

```diff
@@ -46,7 +47,7 @@ def __init__(self, service_name: str, service_spec: serve.SkyServiceSpec,
                                                     spec=service_spec,
                                                     task_yaml_path=task_yaml))
         self._autoscaler: autoscalers.Autoscaler = (
-            autoscalers.RequestRateAutoscaler(
+            autoscalers.HeteroGPUAutoscaler(
```

Comment:
If I understand correctly, this class initialization is incomplete because it's still missing `frequency` and `cooldown` arguments, right?

Reply:
Also I know this isn't prod-ready so it's no big deal, but just wanted to point it out: it's a little confusing that `constants.AUTOSCALER_QPS_WINDOW_SIZE_SECONDS` is passed in for `qps_window_size` and is then immediately overwritten in the autoscaler's `__init__()` method.

Reply:
Thanks for the note! These issues are resolved now